apps/docs/content/components/content/Label.mdx

```diff
@@ -33,16 +33,10 @@ links:
 
 ## Examples
 
-### Sizes
```

Comment:
Is there a reason why we're deleting sizes and inheritance examples?

Reply:
because the Label component is ever only one size. Confirmed with design. This was originally copied from Orbiter.

Reply:
great, thanks for the clarification. We can close this discussion